sdks/typescript/build.sh

```diff
@@ -26,7 +26,7 @@ set -e
 echo "export const version = \"$npm_package_version\";" > src/apache_beam/version.ts
 
 # Using npx to execute ttsc from the local node_modules environment.
-npx ttsc -p .
+npm run tsc
```

Comment:
Can you put a comment in the PR description about what the difference between the two of these are?

Reply:
this is part of switching to https://github.com/nonara/ts-patch, `tsc` is the standard TS compiler